addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -401,6 +386,9 @@ public void packetReceived(byte[] packet) {
                         }
                     }
                 }
+            } catch (IOException e) {
+                logger.error("I/O Error", e);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
```

Comment:
Whether the overflow is less than an error or not is largely a matter of policy. I'd say yes since overflow means that something that was explicitly asked to happen isn't going to and that's the sort of thing that causes angry phone calls so it can't be good. It should be rare since overflowing an unbounded queue is going to be an out-of-memory issue but if someone tries plugging rxcom-bridged devices into their sound-to-light set, well... OTOH 433MHz is unacknowledged and lossy so there's a good argument for warn or info. Maybe not debug though. It _did_ happen and it likely does affect the user experience.